ROCM and HIPIncludePath is now /usr, which disturb the include order
===================================================================
Index: comgr/src/comgr-compiler.cpp
===================================================================
--- comgr.orig/src/comgr-compiler.cpp
+++ comgr/src/comgr-compiler.cpp
@@ -1050,10 +1050,6 @@ amd_comgr_status_t AMDGPUCompiler::addCo
     Args.push_back("-target");
     Args.push_back("x86_64-unknown-linux-gnu");
     Args.push_back("--cuda-device-only");
-    Args.push_back("-isystem");
-    Args.push_back(ROCMIncludePath.c_str());
-    Args.push_back("-isystem");
-    Args.push_back(HIPIncludePath.c_str());
     break;
   default:
     return AMD_COMGR_STATUS_ERROR_INVALID_ARGUMENT;
